src/views/Orders.vue

```diff
@@ -0,0 +1,347 @@
+              </div>
+            </div>
+          </div>
+          <div class="order-items">
+            <ion-card v-for="(item, index) in order.doclist.docs" :key="index" :item="item">
+              <ion-item>
+                <ion-thumbnail slot="start">
+                  <Image :src="getProduct(item.productId).mainImageUrl" />
+                </ion-thumbnail>
+                <ion-label>
+                  <p>{{ item.parentProductName }}</p>
+                  {{ item.productName }}
+                  <p> {{ $t("Color") }} : {{ $filters.getFeature(getProduct(item.productId).featureHierarchy, '1/COLOR/') }} </p>
+                  <p> {{ $t("Size") }} : {{ $filters.getFeature(getProduct(item.productId).featureHierarchy, '1/SIZE/') }} </p>
+                </ion-label>
+                <ion-badge color="success" slot="end"> {{ order.doclist.docs[0].orderItemStatusId }} </ion-badge>
+              </ion-item>
+              <div v-if="false">
+                <ion-item>
+                  <ion-label> {{ $t("Promise date") }} </ion-label>
+                  <p slot="end"> {{ item.promisedDatetime ? $filters.formatUtcDate(item.promisedDatetime, 'YYYY-MM-DDTHH:mm:ssZ') : '-'  }} </p>
+                </ion-item>
+                <ion-item>
+                  <ion-label> {{ $t("PO arrival date") }} </ion-label>
+                  <!-- Need to work on it -->
+                  <p slot="end"> Not available </p>
+                </ion-item>
+                <ion-item>
+                  <ion-label> {{ $t("Last brokered") }} </ion-label>
+                  <!-- Need to work on it -->
+                  <p slot="end"> Not available </p>
+                </ion-item>
+              </div>
+              <div v-else>
+                <ion-item>
+                  <ion-label> {{ $t("Shipping method") }} </ion-label>
+                  <!-- Need to work on it -->
+                  <p slot="end"> Not available </p>
+                </ion-item>
+                <ion-item>
+                  <ion-label> {{ $t("Shipping from") }} </ion-label>
+                  <!-- Need to work on it -->
+                  <p slot="end"> Not available </p>
+                </ion-item>
+                <ion-item>
+                  <ion-label> {{ $t("Location inventory") }} </ion-label>
+                  <!-- Need to work on it -->
+                  <p slot="end"> Not available </p>
+                </ion-item>
+              </div>
+            </ion-card>
+          </div>
+        </div>
+      </div>
+    </ion-content>
+  </ion-page>
+</template>
+
+<script>
+import {
+  IonBackButton,
+  IonBadge,
+  IonButton,
+  IonCard,
+  IonCardContent,
+  IonCheckbox,
+  IonChip,
+  IonContent,
+  IonHeader,
+  IonIcon,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonListHeader,
+  IonNote,
+  IonPage,
+  IonSearchbar,
+  IonSelect,
+  IonThumbnail,
+  IonTitle,
+  IonToolbar,
+  IonSelectOption,
```

Comment:
Arrange the component import alphabetically

Reply:
Arranged Sir